concordium-consensus/src/Concordium/GlobalState/Persistent/LFMBTree.hs

```diff
@@ -5,6 +5,7 @@
 {-# LANGUAGE UndecidableInstances #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
It doesn't seem this is actually needed.

Reply:
It might have been at some version. But true.